3term-study/자연수-뒤집어-배열로-만들기.js

```diff
@@ -0,0 +1,8 @@
+function solution(n) {
+  let answer = [];
+  n = String(n);
+  for (let num of n) {
```

Comment:
```suggestion
  for (let num of String(n)) {
```
```suggestion
const strNum = String(n)
  for (let num of strNum) {
```
기존 변수의 타입을 바꿀 때는 새로운 변수에 담아주거나 변환한 값을 그대로 사용해주세요. (기존 변수에 재 할당하는 방법 X)